src/rustllvm/RustWrapper.cpp

```diff
@@ -411,11 +411,16 @@ extern "C" LLVMRustMetadataRef LLVMRustDIBuilderCreateFunction(
     bool isLocalToUnit,
     bool isDefinition,
     unsigned ScopeLine,
-    unsigned Flags,
     bool isOptimized,
     LLVMValueRef Fn,
     LLVMRustMetadataRef TParam,
     LLVMRustMetadataRef Decl) {
+#if LLVM_VERSION_GE(4, 0)
+  auto Flags = DINode::FlagPrototyped;
+#else
+  unsigned Flags = DINode::FlagPrototyped;
```

Comment:
Can't we avoid these version checks by just unconditionally doing `auto Flags = ...;` (here and in the rest of this file)?